app/pulp/app/serializers/progress.py

```diff
@@ -30,6 +30,11 @@ class ProgressReportSerializer(ModelSerializer):
         read_only=True
     )
 
+    suffix = serializers.CharField(
+        help_text=_("Suffix that will render the progress report."),
```

Comment:
The suffix doesn't do the rendering. Consider replacing with this:  `The suffix to be shown with the progress report`.